vms/platformvm/README.md

```diff
@@ -0,0 +1,21 @@
+# Platform VM
+
+`PlatformVM` is the virtual machine implementing the [P-chain](https://support.avax.network/en/articles/4058243-what-is-the-platform-chain-p-chain) in `avalanchego`.
+
+As such `PlatformVM` is responsible for:
+
+- Tracking permissioned and [elastic](https://medium.com/avalancheavax/banff-elastic-subnets-44042f41e34c) subnets, including the Primary Network subnet, which is defined in P-chain genesis.
```

Comment:
Could we capitalize Subnet and refer to the Primary Network as just the Primary Network instead of Primary Network Subnet?